src/lib.rs

```diff
@@ -1576,6 +1587,7 @@ impl Default for RtcConfig {
             codec_config: CodecConfig::new_with_defaults(),
             stats_interval: Duration::from_secs(1),
             bwe: false,
+            bwe_initial_bitrate: Bitrate::kbps(1500),
```

Comment:
The 300Kbit/s value I used was lifted from libWebRTC somewhere I think, that said since the initial bitrate is quite tied to what you are initially intending to send perhaps we should force the user to always provide it instead?

Reply:
No problem with that. In that case we can use the optional for enabling instead of a separate flag.

Reply:
Based on the other comment thread I think the signature ought to be

```rust
fn enable_bwe(initial_estimate: Bitrate, desired_bitrate: Bitrate)
```

Reply:
I don't think this is needed.
It makes sense to stick to configuration for initial_estimate because it's useful to start in high quality depending on the use case and because it cannot change afterwards (so the api is not misleading). Instead, for something that can be already set otherwise, and is not vital, we already have the setter which is enough.

Reply:
The only problem is that if you don't eventually set the desired bitrate we'll just keep increasing padding with no limit.

Reply:
No you are right, I thought it would pad to the estimate anyway but it only does that if you actually set the desired rate

Reply:
@k0nserv I update making the initial estimate mandatory. See if you're relatively happy with this or still a blocker